content/dogepedia/articles/what-is-a-miner.md

```diff
@@ -5,4 +5,62 @@ date = "2021-10-22"
   name = "Dogecoin"
 +++
 
-A fairly accessible introduction to what a miner is, and the role of miners in the network.
+A **Miner** is a specialized [node](/dogepedia/how-tos/what-is-a-node/) in the Dogecoin network.
+
+After regular [nodes](/dogepedia/articles/what-is-a-node#full-nodes) in the network, by referencing the blockchain, have verified that a Dogecoin transaction is valid, the transaction is shown in a "pending" state. It stays in this state until a **miner** or a collective of miners known as a [**mining pool**](/dogepedia/articles/what-is-a-mining-pool) picks up the transaction.
+
+### Miners: ELI5 {#miners-eli5}
+
+Miners are nodes using special hardware (normally Scrypt ASIC miners) which allows them to solve cryptographic puzzles, in a process referred to as "mining". A miner receives new transactions from the other nodes in the network, combines them in a **block** following predetermined and inviolable rules enforced by the network, and then attempts to solve a puzzle that will generate a special number, called a hash, that is derived by combining the content of the block and a number called nonce. 

```

Comment:
**bold** the word "nonce" and add in parentheses: "(_number used once_)". 

^ Suggest we do this here, and not below, because this is the first occasion in which the term is used.